### PR TITLE
1640 - GIBCT Screenreader bug - School Location table

### PR DESCRIPTION
**Pull request by NateJohnson-BAH**
## Description
The school locations table doesn't announce itself properly to Safari + VoiceOver, and has some invalid HTML. It would be good to refactor this table for assistive device users. Screenshot and code sample below.

As a user, I want the table instructions to match the table, and links be provided if necessary. The sentence mentioning the links in the table has been given conditional rendering logic and will only be displayed when branches exist under the current institution. 

https://github.com/department-of-veterans-affairs/va.gov-team/issues/1640

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/1646

## Testing done
Confirmed locally in UI, via Chrome browser, Apple VoiceOver and Axe accessibility plugin. 

## Screenshots
Errors on page before fix:
Screen Shot 2019-09-10 at 11 50 23 AM

Screenreader and Axe util now approve:
Screen Shot 2019-09-10 at 11 51 14 AM

## Acceptance criteria
- [x] As a VoiceOver user, I want to hear the column header and cell announced properly.
- [x] As a user, I do not want the visual layout to change when the HTML is refactored.

- [x] As a user, I want the table instructions to match the table, and links be provided if necessary.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/components/profile/SchoolLocations.jsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ export class SchoolLocations extends React.Component {
       physicalZip,
     } = institution;
     const nameLabel = this.institutionIsBeingViewed(facilityCode) ? (
-      <h6>{name}</h6>
+      <p className="main-campus-name">{name}</p>
```
Comment (zurbergram) [inline src/applications/gi/components/profile/SchoolLocations.jsx]: this is too specific of a class name as this is being done in the generic `renderRow` function

If this is specific to the Main Campus row changes should either be made in `renderMainRow` or by using css class `${type}-row` which is used by the `<tr>`

```
.main-row {
    h6 {
      margin-bottom: 0em;
      margin-top: 0em;
    }
}
```

Is UI/UX ok with the switch to a `<p>` tag?

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 2]
```diff
@@ -232,16 +232,22 @@
   .school-locations {
     table {
       border: hidden;
-      margin-top: 0em;
-      margin-bottom: 0em;
+      margin-top: 1.5em;
```
Comment (zurbergram) [inline src/applications/gi/sass/partials/_gi-profile-page.scss]: Is UI/UX ok with all of these changes?

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 3]
```diff
@@ -232,16 +232,22 @@
...
     }
 
     th {
       border: hidden;
       background: none;
       white-space: nowrap;
-      padding-bottom: 0em;
+      padding-bottom: 0.5em;
       padding-top: 0em;
+      clear: both;
```
Comment (zurbergram) [inline src/applications/gi/sass/partials/_gi-profile-page.scss]: Is UI/UX ok with all of these changes?

**Review by zurbergram — CHANGES_REQUESTED** [review]
See comments above

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
Approved by QA

**Review by 1Copenut — APPROVED** [review]
I reviewed with axe (Chrome) and VoiceOver (Safari). The table is working as expected with this browser. LMK when this branch is merged into Staging and I'll review with NVDA and JAWS. Should be no issues.

**Review by rianfowler — APPROVED** [review]

**Pull request merged**